example-prev/ios/Podfile

```
@@ -0,0 +1,55 @@
+  use_native_modules!
+end
+
+target 'example-tvOS' do
+  # Pods for example-tvOS
+
+  target 'example-tvOSTests' do
+    inherit! :search_paths
+    # Pods for testing
+  end
+
+end
```

RuboCop (1.81.1)

[convention] Extra empty line detected at block body end.

(Layout/EmptyLinesAroundBlockBody)